infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -73,8 +80,16 @@ func TestSimpleExample(t *testing.T) {
 				annotateGcsFunctionRegion = eventTrigger.Get("triggerRegion").String()
 				fmt.Printf("triggerRegion: %s\n", annotateGcsFunctionRegion)
 			}
+			if strings.Contains(functionName, "functions/" + annotateHttpFunctionName) {
+				reg := regexp.MustCompile(`locations/([^/]+)/`)
+				match := reg.FindStringSubmatch(functionName)
+				if len(match) > 1 {
```

Comment:
suggestion: refactor to avoid nested `if` statements.

Suggestion:

```go
var match []string
var annotateHttpFunctionRegion string
want := fmt.Sprintf("functions/%s", annotateHttpFunctionName)
if strings.Contains(functionName, want) {
    reg := regexp.MustCompile(`locations/([^/]+)/`)
    match = reg.FindStringSubmatch(functionName)
}

if len(match) > 1 {
    annotateHttpFunctionRegion = match[1]
}
```

Reply:
fixed in [c3f370d](https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf/pull/28/commits/c3f370dc3e5ad7baff1afa7e0600ccc24cc4bfc7)